### DISCUSSION TITLE
Easy way to set theme for myst to html conversion?

### DISCUSSION THREAD

**Discussion by fohrloop** [executablebooks/MyST-Parser#781] [Q&A]
First of all, thanks for this wonderful package and the MyST markdown language. 

### Question
I was looking for a simple tool for making a *quick* ("zero config") MyST to HTML conversion, ideally creating a `myfile.html`, and embedding all required css & js to it. Then I found the `myst-docutils-html5` from the [`myst-parser`](https://myst-parser.readthedocs.io/en/latest/) and tested it

```
myst-docutils-html5 myfile.md > myfile.html
```

with 

````markdown
# This is a title

```{code-block} python
def func(x:int) -> int:

    out = 0
    for i in range(x):
        out += x**2
    return out 
```

This is some text

```{note} This is a note
```

(C) Testing Copyright 2023 
````

which gets pretty close but lacks of modern theme:

![image](https://github.com/executablebooks/meta/assets/17479683/38bace9b-c0be-417c-a154-d512e59b9468)

Then I checked the help of `myst-docutils-html5` and it shows:

```
--template=<file>       Template file. (UTF-8 encoded, default:
                        "/path/to/site-packages/docutils/writers/html5_polyglot/template.txt")
```

That `template.txt` then contains

```
%(head_prefix)s
%(head)s
%(stylesheet)s
%(body_prefix)s
%(body_pre_docinfo)s
%(docinfo)s
%(body)s
%(body_suffix)s
```
and there are other files in the folder:

```
📁 html5_polyglot/
├─📄 plain.css
├─📄 math.css
├─📄 __init__.py
├─📄 responsive.css
├─📄 tuftig.css
├─📄 template.txt
└─📄 minimal.css
```

I understand that I could make a theme by copying this folder and editing the contents. But is that really the only way? I see there are nice themes for sphinx. Could these be reused somehow?

**Comment by chrisjsewell** [comment]
> I see there are nice themes for sphinx. Could these be reused somehow?

Heya, yeh this would certainly be ideal, but alas the sphinx themes are currently quite tightly coupled to sphinx and not easy to use for docutils "single page builds" 😢 

Its definitely something I'm interested to look into
